.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
I can understand your reluctance; I am currently partial to the information in the blog post since that provides information on which I can form an opinion. I am unaware of arguments against committing the lock file for a library.

Also, because I wanted to know, the [composer documentation](https://getcomposer.org/doc/02-libraries.md#lock-file) does not provide explicit advice on how to approach this; if anything they seem slightly partial to committing the lock file:

> For your library you may commit the composer.lock file if you want to. This can help your team to always test against the same dependency versions. However, this lock file will not have any effect on other projects that depend on it. It only has an effect on the main project.
>
> If you do not want to commit the lock file and you are using git, add it to the .gitignore.

What is your reasoning for not wanting a lock file in a library?